.github/workflows/gradle-check.yml

```diff
@@ -27,8 +27,28 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v44
+        with:
+          files_ignore: |
+            release-notes/*.md
```

Comment:
Could you please add a comment after each file/entry that justifies why it should be ignored?

If the answer is we want to go fast, why not just have an admin merge bypassing checks?

Reply:
I believe saving time and resources is a generic reason even enough for this change? Running a job for 1+ hour and the backport (1+ hour) which adds no value to the run but can be a blocker due to flaky tests. Bypassing is always an option but usually not an opted one.

Also I am not sure if adding a comment would work or not (needs more testing) as I see one more input `files_ignore_separator` in the action used https://github.com/tj-actions/changed-files?tab=readme-ov-file#inputs-%EF%B8%8F